ACMILabs/transcoder
# Update/5 unified collections exhibitions

*Resolves #5 and #14*

This is a preliminary step towards having a more configurable transcoder that supports both exhibitions and collections video transcoding.
Adds a FLEXIBLE_MASTER_NAMING boolean environment variable in order to support exhibitions and collections file processing from the same codebase.
Adds documentation for deploying to OSX (without balena).
Includes some general refactor and cleanup.
Removes local web copies after copying them to S3.

To test:
1. Set FLEXIBLE_MASTER_NAMING=False
2. Check that a file named like B9234567_mp01_placeholder-2.mp4 transcodes, and that a file named like placeholder-2.mp4 doesn't transcode.
3. Set FLEXIBLE_MASTER_NAMING=True
4. Check that videos transcode with any name, like placeholder-2.mp4
